docs/operator-manual/ingress.md

```diff
@@ -58,6 +58,7 @@ metadata:
     kubernetes.io/ingress.class: nginx
     kubernetes.io/tls-acme: "true"
     nginx.ingress.kubernetes.io/ssl-passthrough: "true"
+    nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
```

Comment:
Are you sure this is required? `ssl-passthrough` usually implies that the backend service is providing a TLS endpoint. At least in my cluster, I don't have to specify that.

Reply:
I don't know. I was try to follow the original docs just changed the host FQDN. But the outcome is browser show error for too many redirect. I try to use cURL to check. It show the return is redirect HTTPS protocol. But I already using HTTPS on browser and cURL.

Reply:
I just remove the ingress and re-create it without `backend-protocol` it still have too many redirect error.
I have to put this annotation to make it work.

Reply:
Do you have another load balancer in front of your Ingress which does not pass SNI to the Ingress? Then nginx-ingress falls back to normal proxy mode IIRC, which could lead to this redirect loop.

Reply:
I have an AWS ELB running in TCP mode with proxy protocol in front of nginx ingress. Since that is TCP mode. That should pass SNI. And other ingress also have TLS enabled, so the SNI should working fine.

Reply:
This was required for webhook delivery to work for us. Without it we always got a 307 response code.

Reply:
How about I comment it out and have a comment message, it user have redirect loop. Need enforce ingress to use HTTPS protocol connect to backend?

Reply:
A comment message would be fine!